bigchaindb_driver/utils.py

```diff
@@ -1,10 +1,12 @@
 """Set of utilities to support various functionalities of the driver.
 
+.. danger:: This module is meant for specific internal usage only and its
+    behaviour is subject to change.
+
```

Comment:
Moved the danger notice to be module-level since I don't think anyone should be using any of this directly.